profile/rwahs.xml

```diff
@@ -8329,6 +8339,13 @@
       <typeRestrictions>
         <restriction type="publicMemorial"/>
       </typeRestrictions>
+      <groupAccess>
```

Comment:
This is copied from [`memorial_object_ui`](https://github.com/kehh/installation-profile/blob/9c15cd22a68ee49d4e5d43d10deae7045426412e/profile/rwahs.xml#L5880-L5886) using the updated `derive-displays.xslt` below.